### PR TITLE
Add support for SENTRY_RELEASE

### PR DESCRIPTION
**Pull request by mtsmfm**
ref: https://github.com/getsentry/sentry-php/issues/804

### CODE REVIEW

File: spec/raven/configuration_spec.rb [hunk 1]
```diff
@@ -126,6 +127,25 @@
...
+    before do
+      allow(File).to receive(:directory?).with(".git").and_return(false)
+      allow(File).to receive(:directory?).with("/etc/heroku").and_return(false)
+    end
+
+    it 'defaults to nil' do
+      expect(subject.release).to eq(nil)
+    end
+
+    it 'uses `SENTRY_CURRENT_ENV` env variable' do
```
Comment (bruno-garcia) [inline spec/raven/configuration_spec.rb]: Test name doesn't match the actual variable name. Could please rename it so they are equal?

**Comment by bruno-garcia** [comment]
One nit. Other than that LGTM.
thanks for contributing!

File: spec/raven/configuration_spec.rb (hunk 1 above)
Comment (mtsmfm) [reply spec/raven/configuration_spec.rb]: Oops 🙈 
Fixed and force pushed

**Comment by bruno-garcia** [comment]
Thanks @mtsmfm !

**Comment by bruno-garcia** [comment]
@mtsmfm FYI we'll make a release on Monday morning UTC+2.

**Pull request merged**